.babelrc

```diff
@@ -1,7 +1,8 @@
 {
   "plugins": [
     "transform-object-assign",
-    "transform-object-rest-spread"
+    "transform-object-rest-spread",
+    "transform-class-properties"
```

Comment:
this plugin is needed to use arrow functions. It was available in the `edx-platform` but not included in the plugins.